JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1 +1,13 @@
-The **DataGrid** widget allows you to&nbsp;customize its toolbar using the **onToolbarPreparing** function. You can add new items to&nbsp;the toolbar or&nbsp;customize the existing ones. In&nbsp;this demo, the toolbar contains a&nbsp;collection of&nbsp;other widgets from the DevExtreme HTML5&nbsp;UI Widgets library.
\ No newline at end of file
+The toolbar contains predefined and custom controls that execute various commands.
```

Comment:
This sentence wouldn't be needed if you accept my correction to the second paragraph. They are almost saying the same thing.

Reply:
Will remove it.